rules/0560-docker_integration_rules.xml

```diff
@@ -13,44 +13,176 @@ ID: 87900 - 87999
     </rule>
 
+    <rule id="87907" level="3">
+        <if_sid>87900</if_sid>
+        <field name="docker.status">^exec_start: </field>
+        <description>Command run in container $(docker.Actor.Attributes.name). Action: '$(docker.Action)'</description>
+        <options>no_full_log</options>
+    </rule>
+
+    <rule id="87908" level="3">
+        <if_sid>87907</if_sid>
+        <field name="docker.status">^exec_start: bash</field>
+        <description>Started shell session in container $(docker.Actor.Attributes.name)</description>
+        <options>no_full_log</options>
+    </rule>
+
+    <rule id="87909" level="3">
+        <if_sid>87900</if_sid>
+        <field name="docker.status">^restart$</field>
+        <description>Container $(docker.Actor.Attributes.name) restarted</description>
+        <options>no_full_log</options>
+    </rule>
+
+    <rule id="87910" level="3">
+        <if_sid>87900</if_sid>
+        <field name="docker.status">^extract-to-dir$</field>
+        <description>Copied a file from host to $(docker.Actor.Attributes.name)</description>
+        <options>no_full_log</options>
+    </rule>
+
+    <rule id="87911" level="3">
+        <if_sid>87900</if_sid>
+        <field name="docker.status">^archive-path$</field>
+        <description>Copied a file from $(docker.Actor.Attributes.name) to host</description>
+        <options>no_full_log</options>
+    </rule>
+
+    <rule id="87912" level="0">
+        <if_sid>87900</if_sid>
+        <field name="docker.Type">^volume$</field>
+        <description>Docker volume action</description>
+        <options>no_full_log</options>
+    </rule>
+
+    <rule id="87913" level="3">
+        <if_sid>87912</if_sid>
+        <field name="docker.Action">^create$</field>
+        <description>Volume created in $(docker.Actor.Attributes.driver)</description>
+        <options>no_full_log</options>
+    </rule>
+
+    <rule id="87914" level="3">
+        <if_sid>87912</if_sid>
+        <field name="docker.Action">^destroy$</field>
+        <description>Volume destroyed in $(docker.Actor.Attributes.driver)</description>
+        <options>no_full_log</options>
+    </rule>
+
+    <rule id="87915" level="3">
+        <if_sid>87900</if_sid>
+        <field name="docker.status">^commit$</field>
+        <description>Container $(docker.Actor.Attributes.name) commited</description>
+        <options>no_full_log</options>
+    </rule>
+
+    <rule id="87916" level="3">
+        <if_sid>87900</if_sid>
+        <field name="docker.status">^untag$</field>
+        <description>Image $(docker.Actor.Attributes.name) untagged</description>
+        <options>no_full_log</options>
+    </rule>
+
+    <rule id="87917" level="3">
+        <if_sid>87900</if_sid>
+        <field name="docker.status">^delete$</field>
+        <description>Container $(docker.Actor.Attributes.name) deleted</description>
+        <options>no_full_log</options>
+    </rule>
+
+    <rule id="87918" level="3">
```

Comment:
Is this rule a duplicate of _87907_?

Reply:
Fixed, now rule 87917 checks if an image is untagged.